rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -26,63 +26,72 @@ Create and document the environment for the automation of processes.
 
-. In the *Process Management* view of the *Process Automation* module, click *Create*.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click *Create*.
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
 ==== Edit a Category
 
 Edit a category to render its name more precisely.
 
-. In the *Process Management* view of the *Process Automation* module, click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
 . Change the name in the *Edit the category* window.
 . Click *OK*.
 
 ==== Remove a Category
 
 Remove categories you no longer need.
 
-Only categories that do not contain processes can be deleted.
+You can only remove categories that do not contain processes.
 
-. In the *Process Management* view of the *Process Automation* module, click the the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the the *Remove* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
 
 === Manage Applications
 
-In the view *Process Applications*, list all applications required for the execution of processes. In the view *Process Management*, assign required applications to processes at process creation or phase transition. In the module *Bot Management*, assign applications to a bot that can execute them. Bots can execute all applications installed on their computer.
+In the view *Process Applications*, you list all applications required for the execution of processes.
+
+In the view *Process Management*, the project manager xref:myrpa-start.adoc#required-process-applications[assigns required applications to a process] at process creation or phase transition.
 
-Applications are not available during down-times or while deactivated. You can deploy a process on a bot which has assigned the required processes. Processes whose applications are currently down are executed to completion and are not restarted until the downtime or deactivation period has ended.
+In the module *Bot Management*, the administrator xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assigns required applications to bots] that can execute them. Bots can execute all applications installed on their computer.
+
+Applications are not available during down-times or while deactivated. A member of the process team can deploy a process on a bot which has assigned the required processes. The bot executes processes whose applications are currently down to completion and restarts them when the downtime or deactivation period has ended.
```

Comment:
What does "down to completion" mean? Does it mean that the apps went down before completion or after completion?

Reply:
Relaced paragraph with:
'Applications are not available during down-times or while deactivated. A member of the process team can deploy a process on a bot that has assigned the required processes. If an application becomes non-available, the bot completes running processes using that application and restarts them when the downtime or deactivation period has ended.'
Is that better understandable?